main.mjs

```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
+import { Next_Stage } from "./src/scripts/stages.mjs";
+
+window.onload = Run_Stage_Required_Scripts();
+const start = document.getElementsByClassName("btn");
+start[0].addEventListener("click", () => {
+  Next_Stage("start");
+});
+const stage1 = document.getElementById("stage_1-btn");
+const stage2 = document.getElementById("stage_2-btn");
+const stage3 = document.getElementById("stage_3-btn");
+const stage4 = document.getElementById("stage_4-btn");
+const prev_stage = document.getElementById("prev_stage");
+
+stage1.addEventListener("click", () => {
+  Next_Stage(1);
+});
+stage2.addEventListener("click", () => {
+  Next_Stage(2);
+});
+stage3.addEventListener("click", () => {
+  Next_Stage(3);
+});
+stage4.addEventListener("click", () => {
+  Next_Stage(4);
+});
+prev_stage.addEventListener("click", () => {
+  Previous_Stage(1);
+});
+//Global vars
+export const totalStages = 4;
+export let stage = 1;

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
export let stage = 1;
```